integration_tests/mockApis/prisonRegister.ts

```diff
@@ -13,11 +13,129 @@ const stubGetPrisonRegister = (): SuperAgentRequest =>
         'Content-Type': 'application/json;charset=UTF-8',
       },
       jsonBody: [
-        { prisonId: 'ALI', prisonName: 'Albany (HMP)', active: false },
-        { prisonId: 'AKI', prisonName: 'Acklington (HMP)', active: false },
-        { prisonId: 'KTI', prisonName: 'Kennet (HMP)', active: false },
-        { prisonId: 'ACI', prisonName: 'Altcourse (HMP)', active: true },
-        { prisonId: 'ASI', prisonName: 'Ashfield (HMP)', active: true },
+        {
+          prisonId: 'ALI',
+          prisonName: 'Albany (HMP)',
+          active: false,
+          male: false,
+          female: false,
+          types: [],
+          addresses: [],
+        },
+        {
+          prisonId: 'AKI',
+          prisonName: 'Acklington (HMP)',
+          active: false,
+          male: false,
+          female: false,
+          types: [],
+          addresses: [],
+        },
+        {
+          prisonId: 'KTI',
+          prisonName: 'Kennet (HMP)',
+          active: false,
+          male: false,
+          female: false,
+          types: [],
+          addresses: [],
+        },
+        {
+          prisonId: 'ACI',
+          prisonName: 'Altcourse (HMP)',
+          active: true,
+          male: true,
+          female: false,
+          types: [
+            {
+              code: 'HMP',
+              description: 'Her Majesty’s Prison',
+            },
+          ],
+          addresses: [
+            {
+              id: 1,
+              addressLine1: 'Brookfield Drive',
+              addressLine2: 'Fazakerley',
+              town: 'Liverpool',
+              county: 'Lancashire',
+              postcode: 'L9 7LH',
+              country: 'England',
+            },
+          ],
+        },
+        {
+          prisonId: 'ASI',
+          prisonName: 'Ashfield (HMP)',
+          active: true,
+          male: true,
+          female: false,
+          types: [
+            {
+              code: 'HMP',
+              description: 'Her Majesty’s Prison',
+            },
+          ],
+          addresses: [
+            {
+              id: 2,
+              addressLine1: 'Shortwood Road',
+              addressLine2: 'Pucklechurch',
+              town: 'Bristol',
+              county: 'Gloucestershire',
+              postcode: 'BS16 9QJ',
+              country: 'England',
+            },
+          ],
+        },
+        {
+          prisonId: 'SKI',
```

Comment:
We also need Stoken ad Leeds in the mock API response because we have integration tests that use these prisons